spec/requests/cars_spec.rb

```diff
@@ -105,4 +105,24 @@
       end
     end
   end
+  
+  path '/cars/{id}/update_status' do
+    put 'Updates the status of a car' do
+      tags 'Cars'
+      parameter name: :id, in: :path, type: :integer, required: true
+      parameter name: :status, in: :query, type: :integer, description: 'New status value (0 or 1)', required: true
+
+      response '200', 'status updated' do
+        run_test!
```

Comment:
can you write some additional validation on the response by passing a block to the `run_test!` method
```
eg:
run_test! do |response|
  data = JSON.parse(response.body)
  expect(data['title']).to eq('foo')
end
```

Reply:
Ref: https://github.com/rswag/rswag?tab=readme-ov-file#the-rspec-dsl